README.rst

```diff
@@ -177,11 +173,25 @@ Full Example
 ------------
 
 To see a simple example, you can download the code `from github
-<http://github.com/mkomitee/wsgi-kerberos>`_. It is in the example directory.
+<http://github.com/deshaw/wsgi-kerberos>`_. It is in the example directory.
 
 Changes
 -------
 
+1.1.0
+`````
```

Comment:
Explicitly mention that the repo has moved and has a new maintainer?

Reply:
Also nice to include the release date corresponding to the version (and you can say “unreleased” here while this is not yet released until the release date is known).

Reply:
> Explicitly mention that the repo has moved and has a new maintainer?

I don't think that's necessary - this detail is not relevant to the module users

Reply:
Certainly the change to being actively maintained again is relevant.

Reply:
Now that you’ve added the “unreleased” to the heading, ReStructured Text is picky and requires you to extend the heading underline at least as far as the heading’s width.

Reply:
@mlucool - thoughts on this?

Reply:
The license change is already mentioned as part of the 1.0.0 release

Reply:
Wait, based on [PyPI](http://pypi.org/project/wsgi_kerberos) and [GitHub](https://github.com/deshaw/wsgi-kerberos/releases) it looks like 1.0.0 has not been released yet. Did you intend to release 1.0.0 already? This accounts for the confusion here. Good the license change is already mentioned in any case.

Reply:
Btw, would be good to include the release date (and in the meantime "unreleased") for the 1.0.0 entry as well.

Reply:
1.0.0 was never released. It just corresponded to a change of license.

Reply:
If there's no intention of releasing 1.0.0, it shouldn't appear in the list of releases in the changelog, and the changes currently listed under 1.0.0 should be merged into the not-yet-released version (currently appears as "1.1.0 (unreleased)").

Otherwise users will assume that 1.0.0 was released and get confused when they see no corresponding release for it on PyPI or GitHub.

Reply:
Ok fair. I'll just combine the 1.1.0 changes into the 1.0.0 release notes and actually release 1.0.0